FossClubWebsite/WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="WebApp_ID" version="3.0">
+  <display-name>ProblemStatement</display-name>
+  <welcome-file-list>
+    <welcome-file>HomePage.html</welcome-file>
+    <welcome-file>index.htm</welcome-file>
+    <welcome-file>index.jsp</welcome-file>
```

Comment:
Read about $routeProvider in angular. You can simply route your pages something like this, 

$routeProvider
    .when("/", {templateUrl: "home.html", controller: "PageCtrl"})
}]);
Whenever there is an action encountered on button, referencing to '/' it will route to the home.html page and loads the PageCtrl controller.